includes/admin/class-pinterest-for-woocommerce-admin.php

```diff
@@ -431,11 +455,22 @@ public function maybe_go_to_service_login_url() {
 				return false;
 			}
 
-			$view = ! empty( $_REQUEST['view'] ) ? sanitize_key( $_REQUEST['view'] ) : null; // phpcs:ignore WordPress.Security.NonceVerification.Recommended --- not needed
+			$context = 'login';
+
+			// phpcs:disable WordPress.Security.NonceVerification.Recommended --- not needed
```

Comment:
Could you explain why the nonce check is not needed here? Skipping a nonce check should have a very good reason why it is unnecessary, and that should also be documented inline as well.

Reply:
Thanks for pointing this out. Checks were ommited from early on here but code has evolved. Added a nonce check.